.github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.yml

```diff
@@ -6,6 +6,7 @@ body:
     attributes:
       label: What is the issue?
       description: |
+       **Users who include information about pirated material will be banned**
```

Comment:
I'd prefer it if this used direct voice instead of passive, and if it came last in the list.

Basically what I described in https://github.com/transmission/transmission/pull/5059#pullrequestreview-1314719241

Reply:
As in, **Users will be banned who include information about pirated material**

Reply:
Hm I think I need to dust off my grammar rules! I think I used the voice terms incorrectly in my last comment. :roll_eyes: 

To be clearer: I meant that I prefer the version in [this comment](https://github.com/transmission/transmission/pull/5059#pullrequestreview-1314719241) as line #4.

> **YOU WILL BE BANNED** if you include information about pirated torrents.

Reply:
Okay.